test/p2p-test.js

```diff
@@ -73,58 +72,6 @@ describe('P2P', function () {
     await node2.close();
   });
 
-  describe('BIP157', function () {
```

Comment:
why did you  move this out to a new file?

Reply:
resolved. Had done it temporarily.